src/lib/features/auth/specs/login-button.spec.ts

```diff
@@ -0,0 +1,11 @@
+import svelte from 'svelte-inline-compile';
+import { LoginButton } from '..';
+import { useLocalVars } from '$lib/shared/lib/jest-hacks';
+import { render } from '@testing-library/svelte';
+
+it('renders the button', async () => {
+  const loginButton = useLocalVars(svelte`<LoginButton />`, [LoginButton]);
```

Comment:
suggestion: no need for `svelte-inline-compile` here, you can just pass the `LoginButton` straight to the `render()` function